Makefile

```diff
@@ -239,6 +240,15 @@ flexdll_initer_gnat.o: flexdll_initer.c
 flexdll_initer_mingw64.o: flexdll_initer.c
 	$(MIN64CC) $(GCC_FLAGS) -c -o $@ $<
 
+build_config.ml: Makefile
+	echo 'let mingw_prefix = "$(MINGW_PREFIX)"' > $@
+	echo 'let mingw64_prefix = "$(MINGW64_PREFIX)"' >> $@
+	echo 'let msvc = "$(subst ",\",$(subst \,\\,$(MSVCC)))"' >> $@
+	echo 'let msvc64 = "$(subst ",\",$(subst \,\\,$(MSVCC64)))"' >> $@
```

Comment:
I see the intention - but, for example, it's not hardened against a path with an apostrophe - which is a legal character in a directory name - while quoting double-quotes, which are not. However - I think `$(notdir $(MSVCC))` gets us enough?